sharktank/tests/models/llama/sharded_llama_test.py

```diff
@@ -5,10 +5,20 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 import unittest
-from typing import Any, Dict, List, Tuple
+from typing import Any, List, Tuple, Union
+import typing
```

Comment:
Import `OrderedDict` from typing above instead of needing the module.

Reply:
Better yet, you can skip the imports for standard collections: https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections. https://peps.python.org/pep-0585/. (Since Python 3.9)

See also https://stackoverflow.com/a/62033243

Reply:
I thought it would clash with the `collections.OrderedDict`, but I guess the one from `typing` is fine for that too.

Reply:
I think you can get by with
```python
from typing import Any, Union
from collections import OrderedDict
```

Then use the standard collection `list, tuple` and `OrderedDict` in your typing hints.